src/components/AuthContainer.js

```diff
@@ -14,6 +14,7 @@ export default Utils.connectAtom(authStore, 'authState')(
       [isSignedIn === undefined && !isPublic, centeredSpinner],
       [isSignedIn === false && !isPublic, h(SignIn)],
       [registrationStatus === undefined && !isPublic, centeredSpinner],
+      [isPublic, children],
```

Comment:
We should double check that this is the behavior we want. I know we deliberately changed this a while back.